client/malicious.js

```diff
@@ -0,0 +1,21 @@
+import ReputationMiningClient from "./main";
```

Comment:
Unrelated to this particular PR I am wondering why the double quotes have made a reappearance here. We should check whether the linting works properly.

Reply:
The linter told us to switch everything to double quotes back in February after upgrading `eslint-colony-config`... https://github.com/JoinColony/colonyNetwork/commit/d00795d5b4c9a1d3a83f410c7bde98da6b0c49a0#diff-fdfdab1ce7fbbccbbf968b92e9f284d2

Reply:
Then I think the linter might not be integrated correctly:
See
https://github.com/JoinColony/eslint-config-colony/blob/master/index.js#L29
and
https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/style.js#L424

Reply:
Yes, it's here:
https://github.com/JoinColony/colonyNetwork/blob/develop/.eslintrc#L15

All the other prettier rules are overwritten by this. If you care about consistency with the other projects this should be

```json
"prettier/prettier": [
  "error",
  {
    "parser": "flow",
    "singleQuote": true,
    "trailingComma": "all",
    "printWidth": 150 // This is the only enforced difference to the other projects
  },
],
```